src/components/Select.jsx

```diff
@@ -1,33 +1,86 @@
 import * as React from "react";
 import Box from "@mui/material/Box";
 import InputLabel from "@mui/material/InputLabel";
+import MenuItem from "@mui/material/MenuItem";
 import FormControl from "@mui/material/FormControl";
 import Select from "@mui/material/Select";
 
 export default function BasicSelect(props) {
+  const { options } = props;
   const [option, setOption] = React.useState("");
 
   const handleChange = (event) => {
     setOption(event.target.value);
+    props.selected(event.target.value);
   };
-  const { optionsGroups } = props;
 
+  const number = options.length;
+  let listOfNames = [];
+  for (let i = 0; i < number; i++) {
```

Comment:
You can use here map function, useful link with example: https://www.freecodecamp.org/news/javascript-map-how-to-use-the-js-map-function-array-method/